src/main/java/com/o19s/es/ltr/query/StoredLtrQueryBuilder.java

```diff
@@ -67,9 +68,17 @@
         declareStandardFields(PARSER);
     }
 
-    public StoredLtrQueryBuilder() {}
-    public StoredLtrQueryBuilder(StreamInput input) throws IOException {
+    public StoredLtrQueryBuilder() {
```

Comment:
I assume that the default empty constructor is needed for some reason, but can't actually be used? And it must not throw an exception at construction time? A comment explaining why we allow building what looks like an unusable StoredLtrQueryBuilder would be welcomed.